shuup/xtheme/layout/_category.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2018, Shuup Inc. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.utils.translation import ugettext_lazy as _
+
+from shuup.xtheme.layout import Layout
+
+
+class CategoryLayout(Layout):
+    identifier = "xtheme-category-layout"
+    help_text = _("Content in this placeholder is shown for this category only.")
```

Comment:
maybe show the category name here, it could be confusing for user to have 100% what category we are talking about as products can have n categories